docs/integrations/webhooks/create-a-webhook.mdx

```diff
@@ -0,0 +1,361 @@
+---
+title: Create a Clerk webhook in your Next.js application
+description: Learn how to create a Clerk webhook with Svix.
+---
+
+# Create a Clerk webhook in your Next.js application
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk + Next.js application",
+      link: "https://clerk.com/docs/quickstarts/nextjs"
+    }
+  ]}
+>
+- Listen for an event in the Clerk Dashboard
+- Create a webhook handler that
+  - uses Svix to verify the webhook signature
+  - receives the webhook's payload
+- Test the webhook locally using ngrok and the Clerk Dashboard
+- Add logic to your application to trigger the webhook
+</TutorialHero>
+
+In this guide, you will learn how to create a Clerk webhook in your Next.js application. You will listen for the `user.updated` event by creating a webhook endpoint in the Clerk Dashboard, creating a webhook handler in your Next.js application, and testing the webhook locally using ngrok and the Clerk Dashboard.
+
+This guide can be adapted to listen for any Clerk event.
+
+## Set up ngrok
+
+To test a webhook locally, you will need to expose your local server to the internet. For this guide, you will use [ngrok](https://ngrok.com/).
+
+1. Go to the [ngrok website](https://ngrok.com/) and create an account.
+2. Once you have made it to the [ngrok dashboard](https://dashboard.ngrok.com/get-started/setup/macos), in the **Installation** section, follow the instructions to install ngrok on your machine.
+3. To deploy your local server to the internet, run the following command in your terminal:
+
+`ngrok http http://localhost:3000`
+
+4. In your terminal, ngrok should display a **Forwarding** URL that looks something like `https://fb3f-2603-1700-2458-3410-a006-dc38-e9d6-1275.ngrok-free.app/`. Copy this URL.
+
+## Create an endpoint in the Clerk Dashboard
+
+To create a webhook endpoint, you must provide the **Endpoint URL** and then choose the events you want to listen to. For this guide, you will listen to the `user.updated` event.
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=webhooks).
+2. In the navigation sidenav, select **Webhooks**.
+3. Select the **Add Endpoint** button.
+4. In the **Endpoint URL** field, paste the ngrok URL you copied earlier followed by `/api/webhooks`. This is the endpoint that you will later create, that Svix will send the webhook payload to. The full URL should look something like `https://fb3f-2603-1700-2458-3410-a006-dc38-e9d6-1275.ngrok-free.app/api/webhooks`.
+5. In the **Message Filtering** section, select the `user.updated` event.
+6. Select the **Create** button.
+7. You will be redirected to your endpoint's settings page. Leave this page open.
+
+## Add your signing secret to your `.env` file
+
+To verify the webhook payload, you will need your endpoint's signing secret. However, you do not want to expose this secret in your codebase, so you will store it in a `.env` file.
```

Comment:
```suggestion
To verify the webhook payload, you will need your endpoint's signing secret. However, you do not want to expose this secret in your codebase, so you will want to provide it as an environment variable. In local development, this can be done by storing the secret in the `.env.local` file.
```

Minor adjustment, as `.env` usually only applies to development. Env vars are handled diferrently in production in most cases.

Reply:
good catch!